### DISCUSSION TITLE
Question on the Astronomical Calculation Window

### DISCUSSION THREAD

**Discussion by psereni** [Stellarium/stellarium#644] [Q&A] [answered]
Dear Sirs/Madams,
I'm facing  following problem: 
I need to know how the visibility of the reference Stars listed in: https://www.eso.org/sci/observing/tools/standards/spectra/stanlis.html
will change during the year from my observation place (Salzburg).

I thought to use the Astronomical Calculation Window and, for example, calculate the Ephemeris over the whole 2019.

However I'm not able to extend the list of objects with whom stellarium will calculate the ephemeris.

Am I doing something wrong?

since I'm performing spectrometric measurement I always need to take some spectrum of one of those referende Stars and for me it would be very useful to know in advance which star is high in the sky during which period of time.

Thank you very much.

Best regards, Paolo Sereni

**Comment by alex-w** [comment, marked as answer]
Seems you need AstroCalc/Graphs tool. The AstroCalc/Ephemeris works for Solar system bodies only.

**Comment by psereni** [comment]
Oh yes. Thank you

**Comment by alex-w** [comment]
Is question still open for you?

**Comment by psereni** [comment]
No thank you, You can close it. Everything is working as expected.

Thanks. Paolo
